### DISCUSSION TITLE
Remove puppet 6.150

### DISCUSSION THREAD

**Discussion by eliassal** [puppetlabs/community#90] [General]
Hi, I have setup puppet 6.15.0 for doing POCs 3 or 4 years ago on a fedora 37 VM. I need to remove it completely but not sure how to do it When googling, all point to uninstall script  puppet-enterprise-uninstaller in folder opt/puppetlabs which is not there.
Thanks for your help

**Comment by eliassal** [comment]
Hi, can somebody please respond to this question?

**Comment by davidsandilands** [comment]
Hi @eliassal is this still an issue for you?
